tests/ragger/test_eip712.py

```diff
@@ -9,19 +9,35 @@
 from ragger.firmware import Firmware
 from ragger.navigator import Navigator, NavInsID
 import json
+from typing import Optional
+from constants import ROOT_SNAPSHOT_PATH
 
 import ledger_app_clients.ethereum.response_parser as ResponseParser
 from ledger_app_clients.ethereum.client import EthAppClient
 from ledger_app_clients.ethereum.eip712 import InputData
 from ledger_app_clients.ethereum.settings import SettingID, settings_toggle
 
 
+class SnapshotsConfig:
```

Comment:
Use a real async in client
Change navigation mechanism when streaming
Get rid of wrapper around navigation

Reply:
Will be looked at in a future PR :+1: